var/spack/repos/builtin/packages/specfem3d-globe/package.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Specfem3dGlobe(AutotoolsPackage):
```

Comment:
Can we inherit from `CudaPackage` and remove the dependency on `cuda` and the `cuda` variant below?

Reply:
I fixed it!